test/expected.txt

```diff
@@ -303,3 +303,4 @@
 || Good bye!
 || ===================================================
 || ===================================================
+
```

Comment:
Does this test fully cover your feature? If it doesn't, it's a good idea to add some unit tests to cover it